commit-message.md

```diff
@@ -0,0 +1,36 @@
+# Guidelines to create great commit messages
+
+
+We use [GitCop](https://gitcop.com) to check that commit messages are
+properly written. The rules are the following:
+
+* The first line of a commit message, called the subject line should
+  not be more than 80 characters long.
+
+* The commit message should end with the following trailers:
+
+  ```
+  Licence: MIT
+  Signed-off-by: User Name <email@address>
+  ```
+
+  where "User Name" is the author's real name and email@address one of
```

Comment:
Don't think we should go the "required signed pgp commit" route just yet. Some day the tooling will be less cumbersome, but not there today. See https://github.com/ipfs/go-ipfs/pull/212#issuecomment-108511260

(but yeah, if people want to sign commits, i won't stop them)